docker/copy-image/action.yaml

```diff
@@ -74,15 +74,18 @@ runs:
       id: image-uri
       shell: bash
       env:
+        SERVICE_NAME: ${{ inputs.service_name }}
         ECR_IMAGE_VERSION: ${{ inputs.ecr_image_version }}
         GHCR_IMAGE_VERSION: ${{ inputs.ghcr_image_version }}
         ACCOUNT_ID: ${{ inputs.aws_account_id }}
       run: |
         echo "ECR_EXISTS=true" >> $GITHUB_ENV
         aws ecr describe-repositories --repository-names $IMAGE_NAME || echo "ECR_EXISTS=false" >> $GITHUB_ENV
-        echo "ECR_IMAGE_URI=$ACCOUNT_ID.dkr.ecr.us-west-2.amazonaws.com/$IMAGE_NAME:$ECR_IMAGE_VERSION" >> $GITHUB_ENV
-        echo "ECR_IMAGE_URI=$ACCOUNT_ID.dkr.ecr.us-west-2.amazonaws.com/$IMAGE_NAME:$ECR_IMAGE_VERSION" >> $GITHUB_OUTPUT
-        echo "GHCR_IMAGE_URI=ghcr.io/$IMAGE_NAME:$GHCR_IMAGE_VERSION" >> $GITHUB_ENV
+        ECR_TRIMMED_IMAGE_VERSION=$(echo $ECR_IMAGE_VERSION | tr / -)
+        GHCR_TRIMMED_IMAGE_VERSION=$(echo $GHCR_IMAGE_VERSION | tr / -)
+        echo "ECR_IMAGE_URI=$ACCOUNT_ID.dkr.ecr.us-west-2.amazonaws.com/$IMAGE_NAME:$SERVICE_NAME-$ECR_TRIMMED_IMAGE_VERSION" >> $GITHUB_ENV
+        echo "ECR_IMAGE_URI=$ACCOUNT_ID.dkr.ecr.us-west-2.amazonaws.com/$IMAGE_NAME:$SERVICE_NAME-$ECR_TRIMMED_IMAGE_VERSION" >> $GITHUB_OUTPUT
+        echo "GHCR_IMAGE_URI=ghcr.io/$IMAGE_NAME:$SERVICE_NAME-$GHCR_TRIMMED_IMAGE_VERSION" >> $GITHUB_ENV
```

Comment:
@WrightTrae does this last line need to be also echoed into `$GITHUB_OUTPUT` ?

Reply:
oh, I see we aren't setting `GHCR_IMAGE_URI` as an output. We should probably start using step outputs in general rather than ENV, as it's easier to trace where variables came from in a workflow. But in any case, I think this is fine given what we're currently doing.

Reply:
@JamesMaroney We aren't using GHCR_IMAGE_URI outside of this action so I decided not to put as an output